### PR TITLE
No CSRF cookie for docs pages

### PR DESCRIPTION
**Pull request by davidfischer**
This marks docs pages served by Django as CSRF exempt. This should prevent a CSRF cookie from being set (it does in dev at least).

### CODE REVIEW

File: readthedocs/core/views/serve.py [hunk 1]
```diff
@@ -144,6 +147,7 @@ def _serve_file(request, filename, basepath):
         return response
 
 
+@csrf_exempt
```
Comment (ericholscher) [inline readthedocs/core/views/serve.py]: We should include a comment here about why we're making these exempt (no cookies on doc pages)

**Comment by ericholscher** [comment]
Removing the form from the 404 page seems like a simple solution, if there's an easy way to do it.

**Comment by davidfischer** [comment]
I removed the language select form from the error pages. This removes setting the cookie.

**Comment by davidfischer** [comment]
I'm not as concerned if corporate hosted docs set a CSRF cookie although it may still be good to look at. Typically, we have a relationship with people visiting corporate docs.

**Pull request merged**